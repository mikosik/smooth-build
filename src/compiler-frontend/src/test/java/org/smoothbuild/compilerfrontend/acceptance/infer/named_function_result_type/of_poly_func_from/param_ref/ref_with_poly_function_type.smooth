myFunc<A>((A)->A param) = param;
