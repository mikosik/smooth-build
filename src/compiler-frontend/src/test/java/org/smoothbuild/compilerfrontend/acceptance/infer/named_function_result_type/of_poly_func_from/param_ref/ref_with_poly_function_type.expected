(A)->A myFunc<A>((A)->A param)
  = param<>;
